lib/screens/customer_product_detail_page.dart

```diff
@@ -126,11 +126,16 @@ class _CustomerProductDetailPageState extends State<CustomerProductDetailPage> {
   Widget _buildStockColorPickerButton(Stock stock, TextStyle textStyle) {
     var _availableStockColors = _getAvailableStockColors(stock);
 
-    _selectedItemColor ??= _availableStockColors.first;
-
     return DropdownButton<ItemColor>(
       value: _selectedItemColor,
       focusColor: paletteBlackColor,
+      hint: Container(
+        padding: const EdgeInsets.symmetric(horizontal: 16),
+        alignment: AlignmentDirectional.centerStart,
+        child: Text('色',
+            style: TextStyle(
+                fontWeight: FontWeight.w300, color: paletteBlackColor)),
```

Comment:
please add commas:
paletteBlackColor **,** ) **,** )

and then run format command on VSCode

On Windows Shift + Alt + F.
On Mac Shift + Option + F.
On Ubuntu Ctrl + Shift + I.